core/src/main/java/org/apache/carbondata/core/util/CarbonUtil.java

```diff
@@ -261,8 +261,6 @@ public static ColumnGroupModel getColGroupModel(int[][] columnGroups) {
     }
     ColumnGroupModel colGroupModel = new ColumnGroupModel();
```

Comment:
This columngroup class can also be removed

Reply:
handled